v2-migration/src/test/java/software/amazon/awssdk/v2migration/ChangeConfigTypesTest.java

```diff
@@ -30,6 +31,8 @@
 import org.openrewrite.test.RecipeSpec;
 import org.openrewrite.test.RewriteTest;
 
+@Disabled("With OpenRewrite version bump, unit tests fail when ExecutionContext#putMessage is used with multiple recipes "
+          + "(invoked in HttpSettingsToHttpClient)")
```

Comment:
You don't have to disable the entire test; you can disable that particular validation step instead through TypeValidation:
https://github.com/openrewrite/rewrite/blob/e84018e2dfc38578eb6c7d5930b6c04002f18627/rewrite-test/src/main/java/org/openrewrite/test/TypeValidation.java#L113-L118

Reply:
Worked, thanks Tim!